Makefile

```diff
@@ -178,10 +177,9 @@ lint:
 	./hack/verify-lint.sh
 
 tidy:
-	@echo "go mod tidy"
-	for dir in . $(GO_SUBMODULE_DIRS); do \
-		cd $$dir && go mod tidy && git diff -U --exit-code go.mod go.sum && cd -; \
-	done
+	go mod tidy && git diff -U --exit-code go.mod go.sum
+	cd pkg/apis && go mod tidy && git diff -U --exit-code go.mod go.sum
+	cd pkg/client && go mod tidy && git diff -U --exit-code go.mod go.sum
```

Comment:
There seems something wrong when execute cmd `cd` in a loop in Makefile :cold_sweat:, so I change it

Reply:
No need to change to the current dir after L181?

Reply:
It works in my VM, but fail in CI